src/bot.ml

```diff
@@ -330,6 +330,26 @@ let callback _conn req body =
                 Server.respond_string ~status:`OK
                   ~body:(f "Received a request to merge the PR.")
                   () )
+              else if
+                string_match
+                  ~regexp:(f "@%s:? [Bb]ench native" @@ Str.quote bot_name)
+                  body
+                && comment_info.issue.pull_request
+                && String.equal comment_info.issue.issue.owner "coq"
+                && String.equal comment_info.issue.issue.repo "coq"
+                && signed
+              then (
+                (fun () ->
+                  action_as_github_app ~bot_info ~key ~app_id
+                    ~owner:comment_info.issue.issue.owner
+                    ~repo:comment_info.issue.issue.repo
+                    (run_bench
+                       ~key_value_pairs:[("coq_native", "yes")]
+                       comment_info ) )
+                |> Lwt.async ;
+                Server.respond_string ~status:`OK
+                  ~body:(f "Received a request to start the bench.")
```

Comment:
Maybe this message could be tweaked to be more specific about the request received?

Reply:
I couldn't even find it in the heroku logs

Reply:
This is not supposed to appear in the Heroku logs. It is the feedback that is sent to GitHub. You can find it by looking at the response of any webhook here: https://github.com/organizations/coq/settings/apps/coqbot-app/advanced